.github/workflows/ci.yml

```diff
@@ -46,3 +48,9 @@ jobs:
         run: |
           npm install
           npm run test
+
+      - uses: codecov/codecov-action@v4
+        with:
+          token: ${{ secrets.CODE_COV }}
+          fail_ci_if_error: true
```

Comment:
What error will make CI failure?

Reply:
this is codecov cli, it depends on network quality. Based on previous experience on teamsfx, this codecov will have more failures caused by network problems. We need to ensure that each PR can submit coverage to codecov, so that the comparison with the base is meaningful.